molgenis-core-ui/src/main/java/org/molgenis/ui/controller/FeedbackController.java

```diff
@@ -51,17 +51,17 @@
 	private final UserService userService;
 	private final AppSettings appSettings;
 	private final CaptchaService captchaService;
-	private final JavaMailSender mailSender;
+	private final Supplier<MailSender> mailSenderSupplier;
```

Comment:
MailSender >> JavaMailSender cause it's way easier to test. I checked and we do not use JavaMailSender capabilities in any of our mail interactions right now so I'd like to switch to the simpler interface.

Will make the bean adhere to MailSender interface and hide the dynamic generation behaviour behind it.